tests/network/vmi_multus.go

```diff
@@ -364,11 +364,12 @@ var _ = SIGDescribe("[Serial]Multus", Serial, decorators.Multus, func() {
 
 				By("Verifying the desired custom MAC is not configured inside the pod namespace.")
 				vmiPod := tests.GetRunningPodByVirtualMachineInstance(vmiOne, testsuite.GetTestNamespace(vmiOne))
+				podInterfaceName := "e31d7ce2712-nic"
```

Comment:
So its basibly checks the external pod interface name (created by the CNI) that was renamed to `...-nic` doesnt have the original mac?

Reply:
yes